Move dataset and filemd5 files alongwith rules

inashivb opened the pull request:
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:
https://redmine.openinfosecfoundation.org/issues/2688
https://redmine.openinfosecfoundation.org/issues/3528

jasonish commented:
The data files seemed to be moved into the correct place. But I'm a little confused why I'm seeing this with the command:
```
./bin/suricata-update -f -D ./data --url https://codemonkey.net/rules.tar.gz
```
this has 3 datafiles that get moved over properly...  But I get this error:
```
18/6/2020 -- 10:17:05 - <Info> -- Backing up current rules.
18/6/2020 -- 10:17:05 - <Info> -- Writing rules to ./data/rules/suricata.rules: total: 348; enabled: 292; added: 348; removed 0; modified: 0
18/6/2020 -- 10:17:05 - <Error> -- filemd5 file fileextraction-chksum.list was not found
18/6/2020 -- 10:17:05 - <Error> -- filesha1 file fileextraction-chksum.list was not found
18/6/2020 -- 10:17:05 - <Error> -- filesha256 file fileextraction-chksum.list was not found
18/6/2020 -- 10:17:05 - <Error> -- filesha1 file missing.list was not found
18/6/2020 -- 10:17:05 - <Info> -- Testing with suricata -T.
```

inashivb commented:
> The data files seemed to be moved into the correct place. But I'm a little confused why I'm seeing this with the command:
> 
> ```
> ./bin/suricata-update -f -D ./data --url https://codemonkey.net/rules.tar.gz
> ```
> 
> this has 3 datafiles that get moved over properly... But I get this error:
> 
> ```
> 18/6/2020 -- 10:17:05 - <Info> -- Backing up current rules.
> 18/6/2020 -- 10:17:05 - <Info> -- Writing rules to ./data/rules/suricata.rules: total: 348; enabled: 292; added: 348; removed 0; modified: 0
> 18/6/2020 -- 10:17:05 - <Error> -- filemd5 file fileextraction-chksum.list was not found
> 18/6/2020 -- 10:17:05 - <Error> -- filesha1 file fileextraction-chksum.list was not found
> 18/6/2020 -- 10:17:05 - <Error> -- filesha256 file fileextraction-chksum.list was not found
> 18/6/2020 -- 10:17:05 - <Error> -- filesha1 file missing.list was not found
> 18/6/2020 -- 10:17:05 - <Info> -- Testing with suricata -T.
> ```

Could you please take a look at #237 ? I was not checking if the rule was enabled before looking for the files.
